### PR TITLE
including advanced net, dhcp and dns policies to automated tests

### PR DESCRIPTION
**Pull request by manrodrigues**

### CODE REVIEW

File: python-test/docs/policies/create_policy_with_dhcp_handler.md [hunk 1]
```diff
@@ -1,12 +1,53 @@
 ## Scenario: Create policy with dhcp handler 
-## Steps:
+## 1 - Create a policy dhcp with description, host specification, bpf filter, pcap source, only qname suffix and only rcode
```
Comment (leoparente) [inline python-test/docs/policies/create_policy_with_dhcp_handler.md:2]: dhcp does not have only qname suffix and only rcode filters

File: python-test/docs/policies/create_policy_with_net_handler.md [hunk 2]
```diff
@@ -1,12 +1,54 @@
 ## Scenario: Create policy with net handler 
-## Steps:
 
-1 - Create a policy with net handler
+## 1 - Create a policy net with description, host specification, bpf filter, pcap source, only qname suffix and only rcode
```
Comment (leoparente) [inline python-test/docs/policies/create_policy_with_net_handler.md:3]: net does not have only qname suffix and only rcode filters. its only for dns

Comment (leoparente) [reply python-test/docs/policies/create_policy_with_net_handler.md:3]: also  "Create a policy net " is a typo.

File: python-test/docs/policies/create_policy_with_dns_handler.md [hunk 3]
```diff
@@ -1,12 +1,54 @@
...
+
+- REST API Method: POST
+- endpoint: /policies/agent/
+- header: {authorization:token}
+
+
+### Expected Result:
+- Request must have status code 201 (created) and the policy must be created
+
+## 4 - Scenario: Create a policy with dns handler,  pcap source, only qname suffix and only rcode
```
Comment (leoparente) [inline python-test/docs/policies/create_policy_with_dns_handler.md:35]: some text lines has double spaces, like here ",  pcap source,"

Comment (manrodrigues) [reply python-test/docs/policies/create_policy_with_dns_handler.md]: nice catch. thanks

**Pull request merged**